src/Types/IDEntityCadastralNumber.php

```diff
@@ -0,0 +1,49 @@
+<?php
+
+declare(strict_types = 1);
+
+namespace AvtoDev\IDEntity\Types;
+
+use Exception;
+use AvtoDev\ExtendedLaravelValidator\Extensions\CadastralNumberValidatorExtension;
+
+/**
+ * Идентификатор - кадастровый номер
+ */
+class IDEntityCadastralNumber extends AbstractTypedIDEntity
+{
+    /**
+     * {@inheritdoc}
+     */
+    public function getType(): string
+    {
+        return static::ID_TYPE_CADASTRAL_NUMBER;
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public static function normalize($value): ?string
+    {
+        try {
+            // Удаляем все символы, кроме разрешенных
+            $value = \preg_replace('~[^\d\:]~u', '', (string) $value);
+
+            //Удаляем первый символ, если это не цифра
```

Comment:
Add ` ` after `//`

Reply:
+ add in IDEntity provider supports for CadastralNumber

Reply:
- `added` single space in comment line
- added in IDEntity